go-controller/hybrid-overlay/pkg/controller/node_linux_test.go

```diff
@@ -33,45 +30,99 @@ import (
 )
 
 const (
-	testOVSMAC     string = "11:22:33:44:55:66"
-	testMgmtMAC    string = "06:05:04:03:02:01"
-	testDRMAC      string = "00:00:00:7a:af:04"
-	testNodeSubnet string = "1.2.3.0/24"
-	testNodeIP     string = "1.2.3.3"
+	testMgmtMAC string = "06:05:04:03:02:01"
+
+	thisNodeSubnet string = "1.2.3.0/24"
+	thisNodeDRIP   string = "1.2.3.3"
+	thisNodeDRMAC  string = "22:33:44:55:66:77"
 )
 
+// returns if the two flowCaches are the same
+func compareFlowCache(returnedFlowCache, expectedFlowCache map[string]*flowCacheEntry) bool {
```

Comment:
@JacobTanenbaum I'd just return the actual errors from this function, then when doing the Ginkgo bits you can:

`Expect(compareFlowCache(...)).NotTo(HaveOccurred())`

and it'll print out the error in the Ginkgo output, as opposed to above in the log output where it might be less visible.

Reply:
done